deploy/swrs/public/table/emission_category.sql

```diff
@@ -5,7 +5,7 @@ begin;
 
 create table swrs.emission_category (
   id integer primary key generated always as identity,
-  swrs_emission_category varchar(1000),
+  swrs_emission_category varchar(1000) unique,
```

Comment:
is the goal of this unique constraint mostly to add an index?

Reply:
Yes